controllers/keda/scaledobject_controller.go

```diff
@@ -19,6 +19,7 @@ package keda
 import (
 	"context"
 	"fmt"
+	"github.com/kedacore/keda/v2/pkg/fallback"
```

Comment:
please move this around lines 46 and properly format

Reply:
ack. done.